src/react/components/connect.tsx

```diff
@@ -3,35 +3,38 @@ import * as React from "react"
 import shallowEqual from "../../utils/shallowEqual"
 import propValidation from "../../utils/propsValidation"
 
-export default function connect(mapToProps) {
-  return Child =>
-    class Connected extends React.Component {
-      static contextTypes = {
-        store: propValidation
-      }
-      unsubscribe
-      state = this.getProps()
-      componentWillMount() {
-        this.unsubscribe = this.context.store.subscribe(this.update)
-      }
-      componentWillUnmount() {
-        this.unsubscribe(this.update)
-      }
-      getProps() {
-        const state =
-          (this.context.store && this.context.store.getState()) || {}
-        return mapToProps(state, this.props)
-      }
-      update = () => {
-        const mapped = this.getProps()
-        if (!shallowEqual(mapped, this.state)) {
-          this.setState(mapped)
-        }
-      }
-      render() {
-        return (
-          <Child store={this.context.store} {...this.props} {...this.state} />
-        )
-      }
+export class Connect extends React.Component<any> {
```

Comment:
What you think about removing the <any>?
Correct me if I'm wrong, but I don't think that it's doing anything here..

Reply:
That's being used to import the Connect component in `src/react/index.tsx`.
https://github.com/concretesolutions/redux-zero/pull/26/files#diff-85c4081ced2fbbdeeed5d03170343962R1

Reply:
Oh wait, I think I misread.  What do you mean by removing the ?

Reply:
I meant removing the "<any">. Sorry about that.

Reply:
TypeScripts linter complains if I remove <any>, any idea how I can get past/around that?

Reply:
Really? Ok, leave it there, in the future I'll try to fix it